src/components/Select/Select.stories.js

```diff
@@ -30,6 +30,14 @@ const SelectTemplate = (args) => ({
     },
   },
 
+  methods: {
+    handleOptionCreated(value) {
+      const newValue = { value, label: value }
+      this.options.push(newValue)
+      this.internalValue.push(value)
+    },
```

Comment:
As the components that use the `SbSelect` are the ones responsible for updating the list and the model with new created options, I've added this small logic to the story to make more sense when testing on the storybook

Reply:
Great!